### ISSUE TITLE
Unnecessary malloc in int to str

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#1775]
The first argument to asprintf is a pointer. I don't think we should allocate something here, right? https://github.com/actonlang/acton/blob/main/base/builtin/int.c#L571

I think this is justa mistake. Right @sydow?

**Comment by sydow** [comment]
Yes, it is certainly a mistake. My guess is that I have made the same mistake at other places in builtin. Will check.
